packages/core/src/codewhisperer/service/serviceContainer.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuthUtil } from '../util/authUtil'
+import { ActiveStateController } from '../views/activeStateController'
+import { LineTracker } from '../tracker/lineTracker'
+
+export class Container {
```

Comment:
How are other developers expected to know what the purpose of this class is?

Reply:
ack, will add it. So my expectation is we need to move other objects into it as well,, e.g. `RecommendationService`, `RecommendationHandler` etc. But that's outside of the scope of this PR and project. We shouldn't add more singleton object if it's not needed.